### ISSUE TITLE
Look Into Having `validate_contents` Always Run with `-v` Flag

### ISSUE THREAD

**Issue by sommersoft** [adafruit/adabot#68]
In talking with @makermelissa on discord about fixing results returned by `validate_core_driver_page`, I ran a fresh report using the `-v "validate_core_driver_page"` flag.

The result had an additional 6 repos listed, compared to a full (non `-v`) run. Those 6 do not have releases on GitHub, so should be ignored. They are listed because `validate_contents` was not run, which didn't trigger adding the un-released repo to [`BUNDLE_IGNORE` list](https://github.com/adafruit/adabot/blob/master/adabot/circuitpython_libraries.py#L522).

All of the validators check if a repo is in the `BUNDLE_IGNORE` list, and skip the check if the repo is in the list.

So, long story short, using the `-v` flag should not produce different results than a full run.
